ukis_pysat/data.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, datahub, catalog=None, url=None):
             self.api = sentinelsat.SentinelAPI(
                 self.user,
                 self.pw,
-                "https://scihub.copernicus.eu/dhus",
+                "https://scihub.copernicus.eu/apihub",

```

Comment:
it is fine, but maybe just delete this line altogether and we are using the [default of sentinelsat](https://github.com/sentinelsat/sentinelsat/blob/master/sentinelsat/sentinel.py#L79), which is what you just changed. That way we can "outsource" this decision.